core/src/test/scala/com/salesforce/op/stages/impl/feature/SmartTextVectorizerTest.scala

```diff
@@ -137,6 +147,39 @@ class SmartTextVectorizerTest
     smart shouldBe expected
   }
 
+  it should "detect one categorical and one non-categorical text feature with html data" in {
```

Comment:
the test is lagerly copy&paste. Can we make it a function or a behavior https://www.scalatest.org/user_guide/sharing_tests

Reply:
I'm not entirely sure this is the right way to test this function either... @mweilsalesforce @Jauntbox @leahmcguire any advice ?

Reply:
look at the text tokenizer test https://github.com/salesforce/TransmogrifAI/blob/master/core/src/test/scala/com/salesforce/op/stages/impl/feature/TextTokenizerTest.scala
it has html stripping tests